docker/arm-server/run.sh

```diff
@@ -0,0 +1,2 @@
+# Shell script to run docker image.
+docker run --privileged -d -P -it cuttlefish-arm64
```

Comment:
does it work even without volume options? run_services.sh expects cf packages.

Reply:
Yes, it works. Uploading & installing debian packages for CF (e.g. cuttlefish-base) is done in build phase. (Defined in Dockerfile) When starting `/run_services.sh`, the packages already lives in the docker image.

Reply:
[this line](https://github.com/google/android-cuttlefish/blob/main/docker/arm-server/run_services.sh#L6) will fail, because there is no /root/cvd_home/bin/cvd in this case, isn't it?

Reply:
Yes, you're right theoretically since it doesn't have cvd binary. But it worked well.

Or, do I remove this shell script? This script isn't used by any operation, just for reference to run docker image.

Reply:
Ah, maybe we could do like this. Running cvd start in /run_services.sh only when cvd binary exists as our intent. Then we could continuously run `build-debs-with-docker.sh` and CF with HO API both.

Reply:
Edited run_services.sh, to be able to run regardless existence of `/root/cvd_home/bin/cvd`.